scrapy/pipelines/images.py

```diff
@@ -88,6 +88,8 @@ def from_settings(cls, settings):
         s3store = cls.STORE_SCHEMES['s3']
         s3store.AWS_ACCESS_KEY_ID = settings['AWS_ACCESS_KEY_ID']
         s3store.AWS_SECRET_ACCESS_KEY = settings['AWS_SECRET_ACCESS_KEY']
+        s3store.POLICY = settings['FILES_STORE_S3_ACL']
```

Comment:
I believe the name should be `IMAGES_STORE_S3_ACL` to be consistent with `IMAGES_STORE`. If so, it should also be documented.